.eslintrc.json

```diff
@@ -14,7 +14,8 @@
   "globals": {
     "flowchart": true, // Blueprints support lib
     "__": true, // Zanata parsing cue
-    "N_": true // Zanata parsing cue
+    "N_": true, // Zanata parsing cue
+    "$window": true
```

Comment:
Ok, that might address the CC error `$window is not defined`  in `client/app/config/authorization.config.js`
What about the same CC error in `client/app/states/logout/logout.state.js`?